Assets/Mirror/Components/Discovery/NetworkDiscoveryBase.cs

```diff
@@ -325,7 +324,7 @@ public void BroadcastDiscoveryRequest()
         /// Override if you wish to include additional data in the discovery message
         /// such as desired game mode, language, difficulty, etc... </remarks>
         /// <returns>An instance of ServerRequest with data to be broadcasted</returns>
-        protected virtual Request GetRequest() => new Request();
+        protected abstract Request GetRequest();
```

Comment:
this looks like its own breaking change, should we return default instead?

Reply:
sure,  default will be fine